src/main/default/lwc/lookup/lookup.js

```diff
@@ -14,9 +14,14 @@ export default class Lookup extends LightningElement {
     @api isMultiEntry = false;
     @api hideResults = false;
     get showResults() {return this.isMultiEntry && !this.hideResults}
-    @api errors = [];
     @api scrollAfterNItems;
     @api iconResources = {};
+    @api errors = [];
```

Comment:
move them back to ehre they were, to reduce the amount of changes we do to this class